src/scripts.js

```diff
@@ -157,3 +160,45 @@ function searchRecipes() {
   displayRecipes(filteredRecipesByName);
   displayRecipes(filteredRecipesByIngredient)
 }
+
+function bindToCookToggleButtons() {
```

Comment:
Clever way to make this work! Good job! Now, we just need to make these recipes display to the page properly!